.github/actions/find-comment/main.js

```diff
@@ -0,0 +1,41 @@
+const github = require('@actions/github');
```

Comment:
I feel like since you're using an external dependency for _updating_ the comment, there was no good equivalent here. Is that a reasonable thing to say?

Reply:
I'm not sure I follow... are you asking if we should use that `peter-evans/create-or-update-comment` action in the workflow because this action is already finding the comment so it may as well update it?

Reply:
Yeah, I just want to know (potentially as a side note in `@fileoverview`) why we have our own. It might not be obvious to an external reader that this is here mostly because we couldn't find a useful external module.

Reply:
yep i'll add a note (that is the reason). We could probably just ditch the other action and write the code ourselves to update or create the comment but it was kind of nice/convenient to just pipe things along to the next workflow step that "does the thing."

Reply:
Hi! I'm the author of `find-comment` and I stumbled across this PR while searching to see if anyone was _actually using_ it. 😆 Have no problem with you hacking on it to build your own, by the way. 👍 

I was just intrigued, though, why the action didn't work for you as-is. It looks like you are searching for the first comment by `github-actions[bot]` in PR comments? I think the action can do that already. Just curious, because if it's something I've missed or a bug I would like to fix it.